Libraries/LibC/netdb.cpp

```diff
@@ -385,15 +394,15 @@ static bool fill_getserv_buffers(char* line, ssize_t read)
 
     __getserv_port_buffer = number.value();
 
-    //Removing any annoying whitespace at the end of the protocol.
+    // Remove any annoying whitespace at the end of the protocol.
     port_protocol_split[1].replace(" ", "", true);
     port_protocol_split[1].replace("\t", "", true);
     port_protocol_split[1].replace("\n", "", true);
```

Comment:
> Is this inplace?


This is weird, newlines can't be here, spaces are replaced with tabs, and the line is split by tabs to get this `port_protocol_split`.